packages/api/models/rules.js

```diff
@@ -87,29 +120,75 @@ class Rule extends Manager {
       throw err;
     }
 
-    const payload = await Rule.buildPayload(item);
```

Comment:
Is there a reason (besides performance) to move this code inside the switch statement?

Reply:
I think, creating a kinesis rule doesn't trigger the workflow event.